python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1062,6 +1062,35 @@ def get_edge_data(self, edge_ids=None, types=None, columns=None):
 
         return None
 
+    def fillna_vertices(self, val=0):
+        """
+        Fills empty vertex property values with the given value, zero by default.
+        Fills in-place.
+
+        Parameters
+        ----------
+        val : object, cudf.Series, or dict
+            The object that will replace "na". Default = 0.  If a dict or
+            Series is passed, the index or keys are the columns to fill
+            and the values are the fill value for the corresponding column.
+        """
+        self.__vertex_prop_dataframe = self.__vertex_prop_dataframe.fillna(val)
```

Comment:
Cool, gotta love simple implementations ;)

You can do `inplace=True` here for both pandas and cudf.  Sorry for being unclear.  Previously, `df[col].fillna(val, inplace=True)` doesn't work for pandas, since it's like `s = df[col] ; s.fillna(val, inplace=True)` where modifying the series inplace doesn't also update the dataframe it's from.

Reply:
right, but `inplace` doesn't work in MG...

Reply:
We often use `inplace` for PropertyGraph but not for MGPropertyGraph, so it's okay to have slightly different implementations.

This was actually one of the first things I asked about when I started working on this, and the reply I got was that `inplace=True` is usually preferred for cudf to be the most efficient.